app/assets/stylesheets/sites/modules/_forms.less

```diff
@@ -131,11 +131,6 @@ input[type="color"] {
     .ie7-inline-block();
   }
 
-  .input-append.color .add-on i {
-    width: 36px;
-    height: 32px;
-  }
```

Comment:
I'm nervous about removing this. It pre-dates Search Consumer by several years (https://github.com/GSA/search-gov/commit/7bd220a1de#diff-ce81856faa309e6c5bfea1aac9c98e1db86d2d4ae4ea48f0cee6772cafd0691cR81), and it looks like it's still in use by the default fonts & colors page: https://github.com/GSA/search-gov/blob/master/app/helpers/affiliate_css_helper.rb#L2